js/loadTrack.js

```diff
@@ -1,24 +1,22 @@
 $('.trackLink').click(function(e) {
-    var link = e.target.dataset.track
-        
-    if  (link === '1' && $('#track1').length === 0) {
-        $('#container').append($('<div>').load('includes/trackLayout1.html'))
-    } else if (link === '1' && $('#track1').length !== 0) {
-        $('.track').css('display', 'none')
-        $('#track1').css('display', 'block')
-    }
+    var trackNo = e.target.getAttribute("data-track"),
```

Comment:
trackNum makes more sense as a variable name

Reply:
also single quotes for 'data-track'

Reply:
🎉